### PR TITLE
Feat/fetching from firebase

### PR DESCRIPTION
**Pull request by saiyedgh**
Let's keep this request open until the changes have been finalized

### CODE REVIEW

**Comment by saiyedgh** [comment]
@jackchinski 

Rendering the three schemas from our `sortedTerms` collection.

You might get 6 console log messages but that will be fixed by removing line 18 of `src/data/sortedTerms.js`

Please have a look at the folder: `src/includes` 
It has all the three schemas that we are rendering on site. Look at the lines of code and manual keying required to render data from each schema. I like `src/includes/Schema01` and `src/includes/Schema03`.

And lastly we can look at the prototypes and then finalize which schema type to stick with.

File: delete.js
Comment (jackchinski) [inline delete.js:1]: pls delete

File: src/App.js [hunk 1]
```diff
@@ -1,63 +1,34 @@
...
-    </div>
+    <BrowserRouter>
+      <div className="main-container">
+        <Header />
+        <Routes>
+          <Route index path="/" element={<Home />} />
+          <Route path="/about" element={<About />} />
+          <Route path="/paste-tos" element={<PasteTOS />} />
+          <Route path="/request" element={<Request />} />
+          <Route path="/terms/:pageName" element={<TermsPage />} /> {/* /terms: termsId */}
```
Comment (jackchinski) [inline src/App.js]: is termsId meant to be uncommented here? if not please remove comment

File: src/includes/Schema01.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                ))}
+              </div>
+            ))}
+          </div>
+        )}
+      </div>
+    );
+  }
+  
+export default Schema01;
```
Comment (jackchinski) [inline src/includes/Schema01.js]: missing line at the end of file

File: src/includes/Schema02.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        }
+        fetchData();
+      }, []);
+  
+    return (
+      <div>
+        {userData ? (
+          <div>
+            <h2>ID: {userData.id}</h2>
+            <h2>Instagram</h2>
```
Comment (jackchinski) [inline src/includes/Schema02.js]: why is Instagram hard coded?

File: src/includes/Schema03.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    }
+    fetchData();
+  }, []);
+
+  return (
+    <div>
+      {userData && (
+        <div key={userData.id}>
+          <h2>{userData.name}</h2>
+          <h2>Twitter</h2>
```
Comment (jackchinski) [inline src/includes/Schema03.js]: same thing with twitter

File: src/includes/components/comingSoon.js
Comment (jackchinski) [inline src/includes/components/comingSoon.js:1]: nice!

File: src/includes/css/footer.css [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+footer nav ul {
+    justify-content: center;
+    padding: 1rem;
+}
+
+footer nav li:not(:last-child)::after {
+    content: "|"; 
+    margin-left: 1rem;
+}
```
Comment (jackchinski) [inline src/includes/css/footer.css]: missing line at end of file

File: src/includes/pages/About.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const About = () => {
+  return (
+    <div>
+      <h1>About Us</h1>
```
Comment (jackchinski) [inline src/includes/pages/About.js]: can we replace the place holder text

File: src/includes/pages/About.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed sit amet urna a odio luctus suscipit. Nullam lacinia ligula nec est sodales, nec vehicula orci lacinia.</p>
+      <p>We are a team of passionate individuals dedicated to providing the best services to our clients.</p>
+    </div>
+  );
+}
+
+export default About;
```
Comment (jackchinski) [inline src/includes/pages/About.js]: also new line

File: src/includes/pages/Home.js
Comment (jackchinski) [inline src/includes/pages/Home.js:1]: very clean i like

**Review by jackchinski — CHANGES_REQUESTED** [review]
Good work, left a couple comments, please address and push again and I'll be ready to approve!

File: src/App.js (hunk 1 above)
Comment (saiyedgh) [reply src/App.js]: Thanks. Resolved

File: src/includes/Schema01.js (hunk 2 above)
Comment (saiyedgh) [reply src/includes/Schema01.js]: Deleted the schemas as we won't be needing them anymore

File: src/includes/Schema02.js (hunk 3 above)
Comment (saiyedgh) [reply src/includes/Schema02.js]: These are old schema testing files. I deleted them.

File: src/includes/Schema03.js (hunk 4 above)
Comment (saiyedgh) [reply src/includes/Schema03.js]: Deleted

File: src/includes/css/footer.css (hunk 5 above)
Comment (saiyedgh) [reply src/includes/css/footer.css]: Formatted the file

**Review by jackchinski — APPROVED** [review]

**Pull request merged**